jerry-core/parser/js/byte-code.h

```diff
@@ -549,6 +549,12 @@
               VM_OC_RESOURCE_NAME) \
   CBC_OPCODE (CBC_EXT_LINE, CBC_NO_FLAG, 0, \
               VM_OC_LINE) \
+  CBC_OPCODE (CBC_EXT_SET_UP_PROPERTY, CBC_HAS_LITERAL_ARG, -1, \
```

Comment:
setup property :)

I would rather use CBC_EXT_SET_STATIC_PROPERTY since it is only used by static functions.